app/controllers/api/mentor_discussion_posts_controller.rb

```diff
@@ -2,10 +2,18 @@ module API
   class MentorDiscussionPostsController < BaseController
     before_action :use_mentor_discussion
 
+    def index
+      posts = @discussion.posts.where(iteration: params[:iteration_id])
```

Comment:
@iHiD For consistency, do we want to use `iteration_id` or `iteration_idx` for this one?

Reply:
(idx as per my earlier comments)